rest-api/etl/setup_local_database_for_etl.sh

```diff
@@ -15,20 +15,23 @@
 # "root" will be used.
 #
 # Make sure the user you run the script as has write access to /var/lib/mysql-files.
+# Generates files under /var/lib/mysql-files/cloud-csv (which can be imported into GCS and Cloud
+# SQL.)
 
 source tools/setup_local_vars.sh
 DB_CONNECTION_NAME=
 CREATE_DB_FILE=/tmp/create_dbs.sql
 CSV_DIR=/var/lib/mysql-files/rdr-csv
-CREDS_FILE=/tmp/rdr_creds.json
+OUTPUT_DIR=/var/lib/mysql-files/rdr-sql-dump
```

Comment:
Does this need to be in `/var/lib`? Generally prefer tmp for this stuff if the idea is short-lived -- but maybe they're being re-used somewhere?

Reply:
Without some config changes to MySQL, yes, you can only access files under /var/lib/mysql-files.

Reply:
The code here is just piping out to this file (mysql is unaware of it). Is it being used somewhere else too?

Reply:
You're right, OUTPUT_DIR can be under /tmp. Changed.